### ISSUE TITLE
training from scratch on custom dataset

### ISSUE THREAD

**Issue by missingstuffedbun** [PointsCoder/VOTR#19]
Your work is inspiring! I was wondering if you can share more tips about training from scratch on custom dataset. THX

**Comment by PointsCoder** [comment]
You may need careful tuning on the RANGE_SPEC and the SIZE hyperparameter in configurations files, which is the most important trick. Training time and learning rate also need carefully tuning, and VoTr generally requires a longer time to converge.
